codegen_glibmm/templates/stub.h.templ

```diff
@@ -55,8 +55,27 @@ protected:
 {% for signal in interface.signals %}
 {% if signal is supported_by_sigc %}
 
-    void {{ signal.name }}_emitter({{ signal.args|map(attribute='cpptype_out')|join(', ') }});
+    void {{ signal.name }}_emitter(const std::vector<Glib::ustring> &destination_bus_names, {{ signal.args|map(attribute='cpptype_out')|join(', ') }}&);
+    inline void {{ signal.name}}_multicastEmitter(const std::vector<Glib::ustring> &destination_bus_names,
+    {%- set comma = joiner() -%}
+    {%- for arg in signal.args -%}
+    {{ comma() }}{{ arg.cpptype_out }} {{ arg.name }}
+    {%- endfor -%}) {
+        {% for arg in signal.args %}
+        {{ signal.name }}_emitter(destination_bus_names, {{ arg.cpptype_to_dbus }}{{ arg.name }});
+        {% endfor %}
+    }
+    inline void {{ signal.name}}_broadcastEmitter(
+    {%- set comma = joiner() -%}
+    {%- for arg in signal.args -%}
+    {{ comma() }}{{ arg.cpptype_out }} {{ arg.name }}
+    {%- endfor -%}) {
+        {% for arg in signal.args %}
+        {{ signal.name }}_emitter({""}, {{ arg.cpptype_to_dbus }}{{ arg.name }});
```

Comment:
I'd rather pass just `{}`, and let the emitter code take care of using an empty destination if an empty list of emitters is passed.

Reply:
I fixed it

Reply:
@mardy 
I try recheck and consider about that, my patch was old ):

{} means to size 0 to vector container,
but, my patch is if "" is added to vector, broadcast emitter called because calling emitter(, null string,) is broadcasting.

so if vector container size is 0 by {}, emitter never called